Add more error messages

cyclotron3k opened the pull request:
PayWay can respond with more errors than were previously in the list.

Reference: https://qsportal.atlassian.net/wiki/display/DOC/Transaction+Response+Codes

jasonwebster commented:
Hi! Thanks for the PR. Do you know if any of these should be mapped to our [standardized error codes](https://github.com/activemerchant/active_merchant/blob/735b0f920e8d17ead9e26c3672cbdf8f7cf1bfb4/lib/active_merchant/billing/gateway.rb#L81-L97)?

cyclotron3k commented:
@jasonwebster I expect so. I'll take a look...

cyclotron3k commented:
I'm closing this pull request because I deleted the fork that it was based on, and was thus unable to make the requested changes. I've re-forked, added all the changes there and made a new pull request: #2333
